rhp/v4/validation.go

```diff
@@ -167,6 +170,9 @@ func (req *RPCRenewContractRequest) Validate(pk types.PublicKey, tip types.Chain
 	// validate the contract fields
 	hp := req.Prices
 	expirationHeight := req.Renewal.ProofHeight + ProofWindow
+	if expirationHeight < hp.TipHeight {
```

Comment:
Probably the answer is no, but should we/could we check the `req.Renewal.ProofHeight` against the `TipHeight` instead? Would be stricter, just not sure if we have to allow renewing/refreshing within the proof window.

Reply:
Actually I noticed we already have that check further down so I reverted this.